src/caches/extrapolation_caches.jl

```diff
@@ -547,11 +568,11 @@ end
 
   # Constant values
   Q::Vector{QType} # Storage for stepsize scaling factors. Q[n] contains information for extrapolation order (n - 1)
-  n_curr::Int64 # Storage for the current extrapolation order
-  n_old::Int64 # Storage for the extrapolation order n_curr before perfom_step! changes the latter
+  n_curr::Int # Storage for the current extrapolation order
+  n_old::Int # Storage for the extrapolation order n_curr before perfom_step! changes the latter
   coefficients::extrapolation_coefficients
-  stage_number::Vector{Int64} # stage_number[n] contains information for extrapolation order (n - 1)
-  sigma::Rational{Int64} # Parameter for order selection
+  stage_number::Vector{Int} # stage_number[n] contains information for extrapolation order (n - 1)
+  sigma::Rational{Int} # Parameter for order selection
```

Comment:
if this shows up in profiles then it should just be changed to a floating point type